terraform/environments/digital-prison-reporting/iam.tf

```diff
@@ -0,0 +1,59 @@
+resource "aws_iam_role" "circleci_iam_role" {
+  name = "circleci_iam_role"
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17",
+    Statement = [{
+      Action = "sts:AssumeRoleWithWebIdentity",
+      Effect = "Allow",
+      Principal = {
+        Federated = aws_iam_openid_connect_provider.circleci_oidc_provider.arn
+      },
+      Condition = {
+        StringLike = {
+          "${aws_iam_openid_connect_provider.circleci_oidc_provider.url}:sub" = "org/${aws_iam_openid_connect_provider.circleci_oidc_provider.url}/*"
+        }
+      }
+    }]
+  })
+}
+
+
+data "aws_iam_policy_document" "circleci_iam_policy" {
+  statement {
+    actions = [
+        "ec2:RunInstances",
+        "ec2:DescribeInstances",
+        "ec2:TerminateInstances",
+        "ec2:CreateTags",
+        "s3:PutObject",
+        "s3:GetObject",
+        "s3:ListBucket",
+        "iam:PassRole",
+        "elasticbeanstalk:*",
+        "rds:*",
```

Comment:
Is this the same as the CICD user policy?  These permissions seem quite open?

Reply:
They're similar, and actually as this is just for one team currently they could be less, it's hard to get exactly what the team require though, will chase them again.